asplib_modules/FFT/CMakeLists.txt

```diff
@@ -21,15 +14,17 @@ file (GLOB_RECURSE ASPLIB_FFT_HEADERS
 
 asplib_source_group("${ASPLIB_FFT_SOURCES}")
 asplib_source_group("${ASPLIB_FFT_HEADERS}")
+
 add_library(asplib_FFT STATIC ${ASPLIB_FFT_SOURCES} ${ASPLIB_FFT_HEADERS})
+add_library(asplib::FFT ALIAS asplib_FFT)
+target_link_libraries(asplib_FFT PRIVATE asplib::Core KissFFT::KissFFT)
+target_include_directories(asplib_FFT
+  PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/FFT
+  INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}
+)
```

Comment:
This might break when used externally through when installed by clients: Search for BUILD_INTERFACE / INSTALL_INTERFACE in https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html